README.md

```diff
@@ -355,13 +355,13 @@ If you have Node v10+ installed locally, you can run the tests, and a local app,
 
 ```sh
 # Install dependencies
-yarn
+yarn install
 
 # Run the tests
-npm test
+yarn test
 
 # Run the app locally
-npm run dev
```

Comment:
This was removed 😅

Maybe remove or suggest `yarn test:watch`

Reply:
@jetersen 

- Fixed in [fdbeae7](https://github.com/release-drafter/release-drafter/pull/1066/commits/fdbeae7af33c8c5f3daa65e925197adabb2baab8)